__tests__/integration.test.js

```diff
@@ -404,4 +404,45 @@ describe('DELETE /api/users', () => {
             })
         })
     })
+})
+
+
+describe('GET /api/articles?topic=', () => {
+    test('GET200: endpoint response is 200 status with an array containing topics matching that query', () => {
+        const topic = 'mitch'
+
+        return request(app)
+        .get(`/api/articles?topic=${topic}`)
+        .expect(200)
+        .then(({ body }) => {
+            const { articles } = body
+
+            expect(articles.length).toBe(12)
+
+            articles.forEach(article => {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: topic,
```

Comment:
I don't think this is any more readable having been extracted out into a variable, but however you want to do it is fine